src/django/rendering.js

```diff
@@ -220,7 +220,7 @@ class Renderer {
 
       if (app == undefined) return
 
-      const models = this.get_models(app_id)
+      const models = this.get_models(app_id).filter(model => !model.abstract)
```

Comment:
So i think at this point we should create a new function
```
get_concrete_models(appid) {
  return this.get_models(appid).filter(model => !model.abstract)
}
```

and use it everywhere you have it, and also on line 223

Reply:
This makes perfect sense, will do!